jdaviz/cli.py

```diff
@@ -144,3 +147,27 @@ def _main(config=None):
     main(filepaths=args.filepaths, layout=layout, instrument=args.instrument, browser=args.browser,
          theme=args.theme, verbosity=args.verbosity, history_verbosity=args.history_verbosity,
          hotreload=args.hotreload)
+
+
+def _specviz():
+    _main(config='specviz')
+
+
+def _specviz2d():
+    _main(config='specviz2d')
+
+
+def _imviz():
+    _main(config='imviz')
+
+
+def _cubeviz():
+    _main(config='cubeviz')
+
+
+def _mosviz():
+    _main(config='mosviz')
```

Comment:
@duytnguyendtn do these need to be changed to use `layout` rather than `config` after your work?

And @maartenbreddels, are these necessary for the stand-alone to work? I changed the standalone entry point to call `main()` instead of `_imviz()`,  but I see they are used in `setup.cfg` although I don't fully understand what those `console_scripts` entry points are doing there.

Reply:
Nope! In the launcher PR, the `config` arg just translates it to layout if provided: https://github.com/spacetelescope/jdaviz/blob/942751809f54a6c8c7abb712f9ea295158bd8c39/jdaviz/cli.py#L146-L149

Reply:
Excellent, got it. I'll just leave this alone then!